docs/configure/environment-variables.md

```diff
@@ -49,25 +68,41 @@ Then you can access this environment variable anywhere, even within your stories
 
 <CodeSnippets
   paths={[
-    'react/my-component-with-env-variables.js.mdx',
-    'react/my-component-with-env-variables.ts.mdx',
-    'vue/my-component-with-env-variables.js.mdx',
-    'vue/my-component-with-env-variables.ts.mdx',
     'angular/my-component-with-env-variables.ts.mdx',
     'web-components/my-component-with-env-variables.js.mdx',
     'web-components/my-component-with-env-variables.ts.mdx',
-    'svelte/my-component-with-env-variables.js.mdx',
-    'solid/my-component-with-env-variables.js.mdx',
-    'solid/my-component-with-env-variables.ts.mdx',
+    'common/my-component-with-env-variables.js.mdx',
+    'common/my-component-with-env-variables.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<IfRenderer renderer={['html', 'react', 'qwik', 'preact','svelte', 'solid', 'vue', 'web-components' ]}>
+
+#### With Vite
+
+Out of the box, Storybook provides a [Vite builder](../builders/vite.md), so using environment variables here will be quite different from using environment variables in other frameworks that use other build tools. To access environment variables in Storybook (e.g., `STORYBOOK_`, `VITE_`), you can use the `import.meta.env`, which is specific to Vite. For example:
```

Comment:
That's not entirely true.  `import.meta` is an ESM standard: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta, and it is extended by Vite (and snowpack, rip), and webpack can also be configured to work that way too: https://www.npmjs.com/package/dotenv-webpack.

```suggestion
Out of the box, Storybook provides a [Vite builder](../builders/vite.md), which does not output Node.js globals like `process.env`. To access environment variables in Storybook (e.g., `STORYBOOK_`, `VITE_`), you can use `import.meta.env`. For example:
```

Reply:
Fixed with the latest commit. Thanks for the suggestion and that bit of information. Appreciate it 🙏